templates/data/custom_dataloader_plugin/upload.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+cd $(dirname $0)
```

Comment:
```suggestion
cd "$(dirname "$0")"
```

this avoids issues with special characters.

Reply:
Fair point, copy pasted it from somewhere else in the repo, will hunt down those examples as well